frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { Link } from 'react-router-dom';
+import { ListGroup } from 'patternfly-react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+import {
+  SidebarSectionHeading,
+  ResourceLink,
+  resourcePath,
+} from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineRunModel, PipelineModel } from '@console/dev-console/src/models';
+import { TopologyOverviewItem } from '../../topology/topology-types';
+import TriggerLastRunButton from './TriggerLastRunButton';
+import { PipelineRunItem } from './PipelineRunItem';
+
+const MAX_VISIBLE = 3;
+
+type PipelinesOverviewProps = {
+  item: TopologyOverviewItem;
+};
+
+const PipelinesOverview: React.FC<PipelinesOverviewProps> = ({
+  item: {
+    pipelines: [pipeline],
+    pipelineRuns,
+  },
+}) => {
+  const {
+    metadata: { name, namespace },
+  } = pipeline;
+  return (
+    <>
+      <SidebarSectionHeading
+        text={PipelineRunModel.labelPlural}
+        style={{ display: 'inline-block' }}
+      />
+      {pipelineRuns.length > MAX_VISIBLE && (
+        <Link
+          className="sidebar__section-view-all"
+          to={`${resourcePath(referenceForModel(PipelineModel), name, namespace)}/Runs`}
+        >
+          {`View All (${pipelineRuns.length})`}
+        </Link>
+      )}
+      <ListGroup componentClass="ul">
+        <li className="list-group-item pipeline-overview">
+          <Flex breakpointMods={[{ modifier: 'justify-content-space-between' }] as any}>
```

Comment:
any is used here to avoid TypeError because breakpoint in `breakpointMods` is not optional and their is no need to use breakpoint with `justify-content-space-between`. Also, Typing for `breakpointMods` suggests `justify-content-flex-space-between` which is invalid and undefined. Correct value is `justify-content-space-between`
https://patternfly-react.surge.sh/patternfly-4/documentation/react/layouts/flex#justify-content-space-between

Reply:
Probably needs an issue over here: https://github.com/patternfly/patternfly-react/issues

We should look to report bugs when we notice them, especially from Patternfly since we use it quite often.

Even when deep-diving into their types I have found:
```
export declare type FlexBreakpointMod = {
  ...
  modifier: 'flex' | ... | 'justify-content-flex-space-between' | ...
  ...
}
```
Which still gives an unfortunate error:
```
ERROR in /Users/andrewballantyne/code/others/console/frontend/packages/dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx(45,36):
TS2322: Type 'string' is not assignable to type '"column" | "flex" | "inline-flex" | "column-reverse" | "row" | "row-reverse" | "nowrap" | "wrap" | "wrap-reverse" | "full-width" | "spacer-none" | "spacer-xs" | "spacer-sm" | ... 44 more ... | "align-content-space-around"'.
```
Not sure what it means by "string" - as they are all strings :/

Their docs don't match up, nor does following the type. Something is up, definitely recommend you log an issue over on their GitHub @sahil143 - let me know if you need anything from me around this.

Reply:
Yes, I will be logging an issue for this. Docs says to use `justify-content-space-between` whereas typing for FlexBreakpointMod suggests `'justify-content-flex-space-between'` which is really weird.
> Not sure what it means by "string" - as they are all strings :/

You might be getting this because you didn't define breakpoint second prop of `FlexBreakpoint Mod`.
Also, the breakpoint prop should be optional here.

Reply:
Nope. I definitely get two different issues with `breakpoint` and `breakpointMods`:

```
ERROR in /.../dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx(46,36):
TS2322: Type 'string' is not assignable to type '"column" | "flex" | "inline-flex" | "column-reverse" | "row" | "row-reverse" | "nowrap" | "wrap" | "wrap-reverse" | "full-width" | "spacer-none" | "spacer-xs" | "spacer-sm" | ... 44 more ... | "align-content-space-around"'.
ERROR in /.../dev-console/src/components/pipelines/pipeline-overview/PipelineOverview.tsx(46,79):
TS2322: Type 'string' is not assignable to type '"sm" | "md" | "lg" | "xl" | "2xl"'.
Version: typescript 3.6.3
```
```
<Flex breakpointMods={[{ modifier: 'justify-content-space-between', breakpoint: 'sm' }]}>
```
Not sure what is up - I'd expect this to work.

Reply:
Yes this will not work because it's expecting `'justify-content-flex-space-between'` instead of `'justify-content-space-between'`
```
<Flex breakpointMods={[{ modifier: 'justify-content-flex-space-between', breakpoint: 'sm' }]}>
```

Reply:
```suggestion
          <Flex breakpointMods={[{ modifier: 'justify-content-flex-space-between' }] as any}>
```
:)

Reply:
`'justify-content-flex-space-between'` only resolves the TypeError, not the actual problem. It should be `justify-content-space-between` in type definitions. `'justify-content-flex-space-between'` is wrong and not defined in patternfly-react styles. It just returns undefined.

Reply:
Please do log an issue with Patternfly @sahil143

Reply:
patternfly issue: https://github.com/patternfly/patternfly-react/issues/3244